src/program.cpp

```diff
@@ -415,6 +415,54 @@ bool spir_binary::load_descriptor_map(
             str << entry;
             cvk_debug("DMAP line: %s", str.str().c_str());
         }
+
+        if (entry.kind == clspv::version0::DescriptorMapEntry::Kind::Sampler) {
+            sampler_desc desc;
+            desc.descriptorSet = entry.descriptor_set;
+            desc.binding = entry.binding;
+
+            desc.normalized_coords = entry.sampler_data.mask & 0x1;
+
+            switch (entry.sampler_data.mask & 0xe) {
```

Comment:
It would be nice to introduce symbolic names for the masks, probably in clspv's `DescriptorMap.h`. They are used in clspv as well.